16-01-2019 10:32:38 info : Starting up the call controller
16-01-2019 10:32:38 debug : [server] , Database URL : mongodb://34.199.158.57:27017/
16-01-2019 10:32:38 info : [app] , Starting up the App
16-01-2019 10:33:08 error : [server] , Error : { MongoNetworkError: failed to connect to server [34.199.158.57:27017] on first connect [MongoNetworkError: connection 0 to 34.199.158.57:27017 timed out]
    at Pool.<anonymous> (/home/development/BitBucket/repo/necti_client_service/Saturn/src/node_modules/mongodb-core/lib/topologies/server.js:562:11)
    at Pool.emit (events.js:159:13)
    at Connection.<anonymous> (/home/development/BitBucket/repo/necti_client_service/Saturn/src/node_modules/mongodb-core/lib/connection/pool.js:316:12)
    at Object.onceWrapper (events.js:254:19)
    at Connection.emit (events.js:159:13)
    at Socket.<anonymous> (/home/development/BitBucket/repo/necti_client_service/Saturn/src/node_modules/mongodb-core/lib/connection/connection.js:256:10)
    at Object.onceWrapper (events.js:254:19)
    at Socket.emit (events.js:159:13)
    at Socket._onTimeout (net.js:411:8)
    at ontimeout (timers.js:478:11)
  name: 'MongoNetworkError',
  message: 'failed to connect to server [34.199.158.57:27017] on first connect [MongoNetworkError: connection 0 to 34.199.158.57:27017 timed out]',
  [stack]: 'MongoNetworkError: failed to connect to server [34.199.158.57:27017] on first connect [MongoNetworkError: connection 0 to 34.199.158.57:27017 timed out]\n    at Pool.<anonymous> (/home/development/BitBucket/repo/necti_client_service/Saturn/src/node_modules/mongodb-core/lib/topologies/server.js:562:11)\n    at Pool.emit (events.js:159:13)\n    at Connection.<anonymous> (/home/development/BitBucket/repo/necti_client_service/Saturn/src/node_modules/mongodb-core/lib/connection/pool.js:316:12)\n    at Object.onceWrapper (events.js:254:19)\n    at Connection.emit (events.js:159:13)\n    at Socket.<anonymous> (/home/development/BitBucket/repo/necti_client_service/Saturn/src/node_modules/mongodb-core/lib/connection/connection.js:256:10)\n    at Object.onceWrapper (events.js:254:19)\n    at Socket.emit (events.js:159:13)\n    at Socket._onTimeout (net.js:411:8)\n    at ontimeout (timers.js:478:11)',
  errorLabels: [ 'TransientTransactionError', [length]: 1 ],
  [Symbol(mongoErrorContextSymbol)]: {} }
